probe-rs/src/config/sequences/esp32c2.rs

```diff
@@ -34,6 +34,8 @@ impl ESP32C2 {
 
 impl RiscvDebugSequence for ESP32C2 {
     fn on_connect(&self, interface: &mut RiscvCommunicationInterface) -> Result<(), crate::Error> {
+        interface.select_hart(0)?;
```

Comment:
I would hope this would be fine as a default, considering the above comments of hart 0 always existing. Do we need to encode this explicitly, and if so, is it really best to do in the chip-specific sequences in code, as opposed to something more generic?